cypress-tests/cypress/e2e/PaymentUtils/Novalnet.js

```diff
@@ -0,0 +1,218 @@
+const successfulNo3DSCardDetails = {
+  card_number: "4200000000000000",
+  card_exp_month: "12",
+  card_exp_year: "25",
+  card_holder_name: "Max Mustermann",
+  card_cvc: "123",
+};
+
+const successfulThreeDSTestCardDetails = {
+  card_number: "4000000000001091",
+  card_exp_month: "12",
+  card_exp_year: "25",
+  card_holder_name: "Max Mustermann",
+  card_cvc: "123",
+};
+
+export const connectorDetails = {
+  card_pm: {
+    PaymentIntent: {
+      Request: {
+        currency: "EUR",
+        customer_acceptance: null,
+        setup_future_usage: "on_session",
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "requires_payment_method",
+        },
+      },
+    },
+    "3DSManualCapture": {
+      Request: {
+        payment_method: "card",
+        billing: {
+          address: {
+              line1: "1467",
+              line2: "CA",
+              line3: "CA",
+              city: "Musterhausen",
+              state: "California",
+              zip: "12345",
+              country: "DE",
+              first_name: "Max",
+              last_name: "Mustermann"
+          },
+          email: "test@novalnet.de",
+          phone: {
+              number: "8056594427",
+              country_code: "+91"
+          }
+        },
+        payment_method_data: {
+          card: successfulThreeDSTestCardDetails,
+        },
+        customer_acceptance: null,
+        setup_future_usage: "on_session",
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "requires_capture",
+        },
+      },
+    },
+    "3DSAutoCapture": {
+      Request: {
+        payment_method: "card",
+        billing: {
+          address: {
+              line1: "1467",
+              line2: "CA",
+              line3: "CA",
+              city: "Musterhausen",
+              state: "California",
+              zip: "12345",
+              country: "DE",
+              first_name: "Max",
+              last_name: "Mustermann"
+          },
+          email: "test@novalnet.de",
+          phone: {
+              number: "8056594427",
```

Comment:
We do not want real phone numbers to be exposed in tests:
```suggestion
              number: "9123456789",
```